mmv1/templates/terraform/examples/gkehub_membership_regional.tf.erb

```diff
@@ -0,0 +1,15 @@
+resource "google_container_cluster" "primary" {
+  name               = "<%= ctx[:vars]['cluster_name'] %>"
+  location           = "us-central1-a"
+  initial_node_count = 1
```

Comment:
In 5.0.0, google_container_cluster.deletion_protection started [defaulting to `true`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/version_5_upgrade#cluster-deletion-now-prevented-by-default-with-deletion_protection) so we need to explicitly set it to `false` here.

If you don't want to include this in the docs, you could use a var to set the value.

```suggestion
  initial_node_count = 1
  deletion_protection = false
```